app/models/spree/tracker.rb

```diff
@@ -14,6 +16,23 @@ def self.current(store = nil)
       Spree::Tracker.where(active: true).joins(:store).where(
         "spree_stores.code = ? OR spree_stores.url LIKE ?",
         store, "%#{store}%"
+      )
+    end
+  end
+
+  def self.by_type(store = nil, type = nil)
+    return if !store
+    if store.is_a?(Spree::Store)
+      Spree::Tracker.where(active: true, store_id: store, tracker_type: type).first
+    else
+      # TODO: Remove in 2.0
```

Comment:
I think the whole content of the else can go away, we don't need to keep deprecated code of other methods, right?

Reply:
done, https://github.com/solidusio-contrib/solidus_trackers/pull/10/commits/66b4108e3cfc0595cad38403e6b2a6c139959cd5#diff-151b7a1ca23474cfb84b18800b5b9957R26